### DISCUSSION TITLE
Get time a job was submitted

### DISCUSSION THREAD

**Discussion by mrwillis** [OptimalBits/bull#2129] [Q&A]
I see the field on Job: `processedOn`. From how I understand it, `processedOn` is when the job was starting processing, i.e., a worker picked it up. Is there a way to get the time when a job was added to a queue? for example, when `queue.add(...)` is called? I can't find a field anywhere for this and it's useful to know the "lag" time between when a job is added and picked up.

**Comment by manast** [comment]
That is the "timestamp" field of the job.
